src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/OO760_add_missing_keys.scala

```diff
@@ -32,27 +32,27 @@ object OO760_add_missing_keys {
 
   val addKontoDatenKey = new Script {
     def execute(sysConfig: SystemConfig)(implicit session: DBSession): Try[Boolean] = {
-      sql"""
+      /*sql"""
 ALTER TABLE KontoDaten ADD KEY id_index (id);
-""".execute.apply()
+""".execute.apply()*/
       Success(true)
```

Comment:
Ganz rauslöschen wenn diese 3 gar nicht mehr genutzt werden?

Reply:
added IF NOT EXISTS check